src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,54 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static int AGE_AT_LEAST_REQUIRE = 18;
```

Comment:
The constant name 'AGE_AT_LEAST_REQUIRE' does not follow the Java naming conventions for constants. It should be in uppercase with underscores, such as 'MIN_AGE_REQUIRED'.